.travis.yml

```diff
@@ -24,3 +24,6 @@ script:
     - make lint
     - make test
     - make coverage
```

Comment:
Do we still need to run `make coverage` in travis if we're having codecov.io build reports?

Reply:
Codecov does not build reports - codecov just presents the reports. You still need to run coverage on your CI machine, so this does need to stay.